examples/apigee-x-vpn-hub-and-spoke-design/apigee_x_project_subnets.tf

```diff
@@ -0,0 +1,120 @@
+##########################################################
+# Copyright 2021 Google LLC.
+# This software is provided as-is, without warranty or
+# representation for any use or purpose.
+# Your use of it is subject to your agreement with Google.
+#
+# Sample Terraform script to set up an Apigee X instance 
+##########################################################
+
+#######################################################################
+### Create 2 subnets, firewall rules and test instances in Apigee X project
+#######################################################################
+resource "google_compute_subnetwork" "apigee_x_project_subnet1" {
```

Comment:
You can use https://github.com/terraform-google-modules/terraform-google-network